DB5/VSTheme.m

```diff
@@ -134,7 +134,19 @@ - (NSImage *)imageForKey:(NSString *)key {
 
 
 - (NSColor *)colorForKey:(NSString *)key {
-
+    // Special overrides for macOS Mojave and beyond
+    if (@available(macOS 10.14, *)) {
+        if ([key isEqualToString:@"dividerColor"]) {
+            return NSColor.separatorColor;
+        } else if ([key isEqualToString:@"tableViewBackgroundColor"]) {
+            return NSColor.controlBackgroundColor;
+        } else if ([key isEqualToString:@"textColor"] || [key isEqualToString:@"noteHeadlineFontColor"]) {
+            return NSColor.textColor;
+        } else if ([key isEqualToString:@"secondaryTextColor"] || [key isEqualToString:@"noteBodyFontPreviewColor"]) {
+            return NSColor.secondaryLabelColor;
+        }
+    }
```

Comment:
Suggestion: Since this is probably bound to grow, maybe move the contents of the if to a method?